prepare.py

```diff
@@ -4,139 +4,144 @@
+    logger.debug("config = %s", config)
 
     bounds = load_bounds()
-    logger.debug('bounds = %s', bounds)
+    logger.debug("bounds = %s", bounds)
 
     if not os.path.exists(opt_base_folder):
-        logger.error('Base path %s does not exist', opt_base_folder)
+        logger.error("Base path %s does not exist", opt_base_folder)
         exit(1)
 
     opt_force = False
-    opt_date = config.get('date')
-    extents = config.get('extents')
+    opt_date = config.date
+    extents = config.extents
+    # TODO(MB) Update this to use argparser module
     for o, a in opts:
-        if o == '-b' or o == '--bounds':
+        if o == "-b" or o == "--bounds":
             try:
                 extents = bounds[a]
             except:
-                logger.error('Invalid bounds %s', a)
-                logger.error('Available bounds ->\n%s',
-                             '\n'.join([a for a in bounds.keys()]))
+                logger.error("Invalid bounds %s", a)
+                logger.error(
+                    "Available bounds ->\n%s", "\n".join([a for a in bounds.keys()])
+                )
             continue
-        if o == '-F' or o == '--force':
+        if o == "-F" or o == "--force":
             opt_force = True
             continue
-        if o == '-d' or o == '--date':
+        if o == "-d" or o == "--date":
             try:
                 opt_date = date.fromisoformat(a)
             except:
-                logger.error('Invalid date %s', a)
+                logger.error("Invalid date %s", a)
                 opt_date = None
             continue
         assert False, "Unhandled option"
 
-    input_dir = os.path.join(opt_base_folder, 'input')
-
-    filtered_graph_folder = os.path.join(opt_base_folder, 'graphs', 'filtered')
+    filtered_graph_folder = os.path.join(opt_base_folder, "graphs", "filtered")
 
     if not opt_date:
-        logger.error('No date provided')
+        logger.error("No date provided")
         usage()
-    logger.debug('opt_date is %s', opt_date)
+    logger.debug("opt_date is %s", opt_date)
 
-    date_filter_string = '{}:{}'.format(opt_date, opt_date + timedelta(days=1))
-    logger.debug('date_filter_string is %s', date_filter_string)
+    date_filter_string = "{}:{}".format(opt_date, opt_date + timedelta(days=1))
+    logger.debug("date_filter_string is %s", date_filter_string)
 
     if not extents:
-        logger.error('No extents provided')
+        logger.error("No extents provided")
         usage()
-    logger.debug('Extents set to %s', extents)
+    logger.debug("Extents set to %s", extents)
 
     if opt_force:
         shutil.rmtree(filtered_graph_folder, ignore_errors=True)
 
     if os.path.exists(filtered_graph_folder):
         logging.warning(
-            'A folder of filtered GTFS and OSM files already exists. To filter again, delete this folder.')
+            "A folder of filtered GTFS and OSM files already exists. To filter again, delete this folder."
```

Comment:
Or, re-run prepare.py with force overwrite [-F | --force] enabled?

Reply:
Fixed in 73014cf3fff0ce7ecfc1b0144123aafe469d019c